src/common/config/index.ts

```diff
@@ -1,13 +1,17 @@
 /*eslint-disable */
-const MAP_SERVER = 'http://10.28.11.125';
-const PUBLISH_POINT = 'blue_m_flat2d-v001';
-const CHANNEL = 1002;
-const VERSION = 1;
-const REQUEST = 'ImageryMaps';
+const MAP_SERVER = (window as any)._env_.MAP_SERVER;
```

Comment:
Multiple user of (window as any)._env_. Maybe assign to a new constant for readability?

Reply:
What do you mean?